cstrike/addons/amxmodx/scripting/CA_Gag.sma

```diff
@@ -1455,11 +1455,10 @@ static LoadConfig() {
     g_gagReasonsTemplates_size = 0
   }
 
-  AutoExecConfig(true, "CA_Gag", "ChatAdditions")
```

Comment:
https://github.com/alliedmodders/amxmodx/blob/e6d9d9b71c36eef81f169698847cc0af44c69b34/plugins/include/amxmodx.inc#L3422-L3424

Are we refusing to create a config?

Reply:
Ops. My bad. Fixed